feat(keepalived_kube): support for IPv6 VIP (#2)
stuvusIT/ansible_keepalived_kube: README.md

# Keepalived

> This role configures keepalived as a static Pod on a Kubernetes master node. It supports either a IPv4  virtual IP or IPv4 and IPv6 simultaneously.

Reviewer: New sentence should be on a new (markdown source) line.

## Requirements

This role needs the host to be configured as a working Kubernetes master node.

## Role Variables

| Name                              |     Required/Default      | Description                                                                                               |
| --------------------------------- | :-----------------------: | --------------------------------------------------------------------------------------------------------- |
| `keepalived_kube_interface`       |    :heavy_check_mark:     | The interface on which to configure the VIP                                                               |

> | `keepalived_kube_vip_v4`          |    :heavy_check_mark:     | The virtual IPv$ address to use                                                                           |

Reviewer: Typo

```suggestion
| `keepalived_kube_vip_v4`          |    :heavy_check_mark:     | The virtual IPv4 address to use                                                                           |
```

Reviewer: Also, should this really be required, or should we make it optional, such that one can choose to only configure IPv6?

| `keepalived_kube_vip_v6`          |        `undefined`        | The virtual IPv6 address to use.                                                                          |
| `keepalived_kube_image`           |    `bsctl/keepalived`     | image to use for the keepalived container                                                                 |
| `keepalived_kube_namespace`       |       `kube-system`       | The namespace of the pod                                                                                  |
| `keepalived_kube_router_id_v4`    |           `51`            | Router id of the vrrp instance responsible for the IPv4 IP. Should be shared by all hosts in the cluster. |

> | `keepalived_kube_router_id_v6`    |           `52`            | Router id of the vrrp instance responsible vor the IPv6 IP. Should be shared by all hosts in the cluster. |

Reviewer: Typo: vor should be for.

```suggestion
| `keepalived_kube_router_id_v4`    |           `51`            | Router ID of the vrrp instance responsible for the IPv4 IP. Should be shared by all hosts in the cluster. |
| `keepalived_kube_router_id_v6`    |           `52`            | Router ID of the vrrp instance responsible for the IPv6 IP. Should be shared by all hosts in the cluster. |
```

# Keepalived

This role configures keepalived as a static Pod on a Kubernetes master node.

> It supports an IPv4  virtual IP as well as an IPv6 one.

Reviewer: One whitespace too much.

```suggestion
It supports an IPv4 virtual IP as well as an IPv6 one.
```
